### PR TITLE
SpaceStatus stuff (Issue #84)

### PR DESCRIPTION
**Pull request by Vertaalfout**
Implemented neutral asb UI, proper text for ASB variations within fleet space, SpaceStatus::OnRender callback

### CODE REVIEW

File: libzhlgen/test/functions/ELF_amd64/1.6.13/MouseControl.zhl [hunk 1]
```diff
@@ -20,6 +20,8 @@ cleanup __amd64 void MouseControl::SetTooltip(MouseControl *this, const std::str
 cleanup __amd64 void MouseControl::SetTooltipTitle(MouseControl *this, const std::string& tooltip);
 ".554889fd5389f34883ec0883fe01":
 cleanup __amd64 void MouseControl::SetDoor(MouseControl *this, int state);
+"415541544989fc55534889f34883ec38":
```
Comment (Nasa62) [inline libzhlgen/test/functions/ELF_amd64/1.6.13/MouseControl.zhl:23]: Signature needs to be relative

File: libzhlgen/test/functions/ELF_amd64/1.6.13/SpaceStatus.zhl [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"415741564155415449bc0000":
+cleanup __amd64 void SpaceStatus::RenderWarningText(SpaceStatus *this, int effect, int textOffset);
+"415455534889fb4883ec10488b7f58e8":
+cleanup __amd64 void SpaceStatus::OnRender(SpaceStatus *this);
+"41544189d45589f5534889fb4883ec20":
```
Comment (Nasa62) [inline libzhlgen/test/functions/ELF_amd64/1.6.13/SpaceStatus.zhl:5]: All new 64-bit signatures except the one at the top of the file should be relative if possible

Comment (Nasa62) [reply libzhlgen/test/functions/ELF_amd64/1.6.13/SpaceStatus.zhl:5]: Also I have not validated these signatures yet

File: FTLGameELF64.h
Comment (Nasa62) [inline FTLGameELF64.h:1]: Why is ELF64 missing the changes to SpaceStatus that Win & ELF32 have?

File: libzhlgen/test/functions/ELF_amd64/1.6.13/SpaceStatus.zhl [hunk 3]
```diff
@@ -0,0 +1,8 @@
+"41564889d08d4add48c1e820"
+cleanup __amd64 void SpaceStatus::OnInit(SpaceStatus *this, SpaceManager *space, Point pos);
+".415741564155415449bc0000":
+cleanup __amd64 void SpaceStatus::RenderWarningText(SpaceStatus *this, int effect, int textOffset);
+".41544189d45589f5534889fb4883ec20":
+cleanup __amd64 void SpaceStatus::MouseMove(SpaceStatus *this, int mX, int mY);
+".415455534889fb4883ec10488b7f58e8":
+cleanup __amd64 void SpaceStatus::OnRender(SpaceStatus *this);
```
Comment (Nasa62) [inline libzhlgen/test/functions/ELF_amd64/1.6.13/SpaceStatus.zhl:8]: missing newline at end of file.

File: libzhlgen/test/functions/ELF_x86/1.6.13/SpaceStatus.zhl [hunk 4]
```diff
@@ -0,0 +1,8 @@
+"55575631f65383ec7c8b9c24????????":
+cleanup __cdecl void SpaceStatus::OnInit(SpaceStatus *this, SpaceManager *space, Point pos);
+"5589e557565381ec9c0000008b55088b":
+cleanup __cdecl void SpaceStatus::RenderWarningText(SpaceStatus *this, int effect, int textOffset);
+"57565383ec208b5c24308b432c890424":
+cleanup __cdecl void SpaceStatus::OnRender(SpaceStatus *this);
+"57565383ec208b5c24308b7424348b43":
+cleanup __cdecl void SpaceStatus::MouseMove(SpaceStatus *this, int mX, int mY);
```
Comment (Nasa62) [inline libzhlgen/test/functions/ELF_x86/1.6.13/SpaceStatus.zhl:8]: Missing newline at end of file.

File: libzhlgen/test/functions/win32/1.6.9/SpaceStatus.zhl [hunk 5]
```diff
@@ -0,0 +1,8 @@
+"578d????0883e4??ff????5589e557565389cb89de81ec????????":
+__thiscall void SpaceStatus::OnInit(SpaceStatus *this, SpaceManager *space, Point pos);
+"578d????0883e4f0ff????5589e557565381ecac0000008b7160":
+__thiscall void SpaceStatus::RenderWarningText(SpaceStatus *this, int effect, int textOffset);
+"578d????0883e4f0ff????5589e557565389cb83ec2c8b492c":
+__thiscall void SpaceStatus::OnRender(SpaceStatus *this);
+"578d????0883e4f0ff????5589e557565389cb83ec4c8b??8b4130":
+__thiscall void SpaceStatus::MouseMove(SpaceStatus *this, int mX, int mY);
```
Comment (Nasa62) [inline libzhlgen/test/functions/win32/1.6.9/SpaceStatus.zhl:8]: Missing newline at end of file.

**Review by Nasa62 — DISMISSED** [review]
(I did not validate signatures as part of this current review)

Biggest concern is the missing SpaceStatus changes in ELF64

**Review by ChronoVortex — APPROVED** [review]

**Pull request merged**